src/main/java/com/example/cecv_e_commerce/domain/dto/order/OrderRequestDTO.java

```diff
@@ -2,5 +2,16 @@
 
 import java.util.List;
 
-public record OrderRequestDTO(List<OrderItemRequestCreateDTO> orderItems) {
-}
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+
+public record OrderRequestDTO(
+    @NotNull(message = "Order items must not be null")
+    @Valid
+    List<OrderItemRequestCreateDTO> orderItems,
+
+    @NotNull(message = "Order shipping must not be null")
+    @Valid
+    OrderShippingRequestCreateDTO orderShipping
+) {}
+
```

Comment:
```suggestion
) {
}
```
khuyến khích nên ri cho đồng nhất và dễ đọc code cu nha